staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5475,7 +5475,7 @@ type ReplicationControllerSpec struct {
 	// Label keys and values that must match in order to be controlled by this replication
 	// controller, if empty defaulted to labels on Pod template.
 	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
-	// +optional
```

Comment:
Currently the options here would likely be:
1. Mark this as +k8s:required to align the DV validation logic with the hand-written validation BUT leave +optional (for Open API backwards compatibility)
2. Change this to be +required.

I am going to reach out to others to understand more about the best option here